.changeset/tricky-glasses-try.md

```diff
@@ -0,0 +1,6 @@
+---
+"@graphiql/react": patch
+"graphiql": patch
+---
+
+Adds a box-model reset for all children of the .graphiql-container class. This change facilitated another change to the --sidebar-width variable. 
```

Comment:
```suggestion
Adds a box-model reset for all children of the `.graphiql-container` class. This change facilitated another change to the `--sidebar-width` variable. 
```

Reply:
This bit _is_ meant to be human readable...good catch!